src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/ccd/State.java

```diff
@@ -7,6 +7,7 @@ public enum State {
 
     APPEAL_STARTED("appealStarted"),
     APPEAL_SUBMITTED("appealSubmitted"),
+    PENDING_PAYMENT("pendingPayment"),
```

Comment:
The new flow transition is: 'Appeal Started' -> 'Appeal Submitted' and finally 'Appeal Paid'
The 'Pending Payment' state is already inferred from the 'Appeal Submitted' status, right?

Reply:
The flow isn't really a factor in this PR, as this PR deals with the update of the "paymentStatus" field in the case data. 
As things currently are, the case is in the "pendingPayment" state when the paymentStatus update happens, so I needed to add this event to the list or else it'd throw an error when parsing a json in between calls. If necessary it can be amended in the next PR whenever we need to reorganize the states, if we see that the case will be in a different state when paying. But this PR isn't meddling with the state and events order.